app/views/users/show.html.erb

```diff
@@ -1,7 +1,9 @@
 <h3>Uploaded by <%= link_to user.name, user %>:</h3>
+<p><%= user.name.capitalize %> has uploaded <%= user.notes.where(created_at: (Time.now - 7.days)..Time.now).count %> notes and <%= user.photos.where(created_at: (Time.now - 7.days)..Time.now).count %> photos in the last week<p>
```

Comment:
These `where(created_at: (Time.now - 7.days)..Time.now).count` queries should be in the user model.